src/components/ScrollToTop.js

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from "react";
+import { useLocation } from "react-router-dom";
+
+export default function ScrollToTop() {
```

Comment:
When you navigate to different pages the scroll position stays the same. This is a helper to scroll to top when navigating